Add docs for SNMP profiles constant metrics (#14413)
DataDog/integrations-core: docs/developer/tutorials/snmp/profile-format.md

          extract_value: '([a-zA-Z0-9_]+)' # will ignore surrounding non-printable characters
```

> #### Report states

Reviewer: ```suggestion
#### Apply mapping to OID value in `metric_tags`
```

I think we can be more generic here. Mapping `states` is only one specific case, but we can map any value to another value.

Author: My goal was to document how to use constant metrics to report string OIDs, `Apply mapping to OID value in metric_tags` is already documented [here](https://github.com/DataDog/integrations-core/blob/9259684c3047a7100f6f17aaee27cb6820895e75/docs/developer/tutorials/snmp/profile-format.md?plain=1#L278-L301) I think
But I agree the title is not generic enough, maybe something like `Report string OIDs`?

> In some cases, you may want to report statuses from your network devices. To do so, you can use the constant metrics feature available in Agent 7.45+.
> 
> `constant_value_one` will send a constant metric equal to one, that can be tagged by string properties.
> 
> An example use case :

Reviewer 2: ```suggestion
To report statuses from your network devices, you can use the constant metrics feature available in Agent 7.45+.

For example: `constant_value_one` sends a constant metric, equal to one, that can be tagged with string properties.

Example use case:
```

```yaml
metrics:
  - MIB: MY-MIB
    symbols:
      - name: myDevice
        constant_value_one: true
    metric_tags:
      - tag: status
        column:
          OID: 1.2.3.4
          name: myStatus
        mapping:
          1: up
          2: down
    # ...
```

> A `snmp.myDevice` metric will be sent, with a value of 1 and tagged by statuses. This will allow to monitor status changes, number of devices per state, ... in the Datadog app.

Reviewer 2: ```suggestion
An `snmp.myDevice` metric is sent, with a value of 1 and tagged by statuses. This allows you to monitor status changes, number of devices per state, etc., in Datadog.
```
